src/todoList.viewModel.js

```diff
@@ -3,10 +3,20 @@ import {Todo} from './todo.model.js';
 
 export class ViewModel {
     constructor() {
-        this.todoList = [new Todo("Try javaScript"), new Todo("Buy a Unicorn")]        
+        this.todoList = ko.observableArray([]);        
+        this.inputText = ko.observable("");
     }
 
     toggleChecking(todo) {
         todo.checked(!todo.checked());
     }
+
+    addNewTodo(todo, event) {
+        if(event.keyCode === 13 && this.inputText() !== '') {
```

Comment:
Define a constant for 13. Right now it is unclear what is it's purpose (magical number)